### PR TITLE
Adding new components and refactoring

### PR DESCRIPTION
**Pull request by iansolano**
### Status
**READY**

### Description
**FEATURE/REFACTOR**

Adding components and refactoring styles to support tutor payments work.

### Related PRs
https://github.com/hiclark/tutor-payments/pull/238

### Quality Checklist:
These are mandatory and must be completed before merging the PR

- [x] New, changed or refactored functionality has tests.
- [ ] New, changed or refactored functionality has documentation.
- [x] New, changed or refactored functionality has been QA'd.
- [x] New, changed or refactored functionality has been added to the CHANGELOG.

### CODE REVIEW

File: src/button/styles.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+const MAX_WIDTH = '17.5rem';
+
+const primary = css`
+  max-width: ${MAX_WIDTH};
+  width: auto;
+`;
+
+const secondary = css`
+  max-width: 100%;
+  width: 100%;
```
Comment (Jexeones24) [inline src/button/styles.js]: Why do we need both if they are the same?

File: src/button/styles.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+const outline = disabled => css`
+  background: ${disabled ? GREY_25 : WHITE};
+  border: ${disabled
+    ? `${BW_1} solid ${GREY_25}`
+    : `${BW_1} solid ${CLARK_PRIMARY}`};
+  color: ${disabled ? WHITE : CLARK_PRIMARY};
+
+  &:hover {
+    border: ${BW_1} solid transparent;
+    color: ${disabled ? WHITE : WHITE};
```
Comment (Jexeones24) [inline src/button/styles.js]: Is this right?

File: src/button/styles.js (hunk 1 above)
Comment (iansolano) [reply src/button/styles.js]: shouldn't

**Comment by iansolano** [comment]
Button is a component that really needs and will get a totally refactor. As I am addressing your feedback @Jexeones24 I am coming across lots of issues but going to leave alone for now since we can tackle later when we refactor to accommodate new styles etc (very soon).

**Comment by iansolano** [comment]
@Jexeones24 good to go?

**Pull request merged**